### ISSUE TITLE
Auto Update wipes MQTT Settings

### ISSUE THREAD

**Issue by spelech** [ESPresense/ESPresense#90]
**Describe the bug**
After enabling auto update (and receiving one), the ESP32s no longer connect to local mqtt instance as previously set.  Resets to mqtt.z13.org 1883

**Expected behavior**
Preserve MQTT settings just like wifi persists through upgrade

**Hardware Details**
ESP32 Dev modules (Micro Center/Inland branded)

This might be partially my own doing but something worth double checking.  I previously was using platformio/vs code to build and upload via usb to the boards.  I prepopulated defaults.h with my relevant settings, so never actually manually set the mqtt settings via the web portal (but did use the web portal for wifi config).

**Comment by DTTerastar** [comment]
Yeah, if you used the .h file auto update will do exactly that by design.
